code/modules/projectiles/guns/projectile/saw.dm

```diff
@@ -10,7 +10,6 @@
 	weapon_weight = WEAPON_MEDIUM
 	fire_sound = 'sound/weapons/Gunshot3.ogg'
 	var/cover_open = 0
-	can_suppress = 0
```

Comment:
How on earth can you suppress a SAW

Reply:
I was going to comment on that also. Does seem a bit odd but I know little about guns IRL.

Reply:
Googled it. Videos of SAWs destroying surpressers.

Reply:
Suppressors work fine on SAWs for a little while, but they will shred apart when the barrel heats up. Can you add a way to make it break after X number of shots?

Reply:
It's definitely possible but I'm not sure if it's worth it.

Reply:
Most guns can be suppressed. The Saw is an open bolt LMG, but it strips cartrages from the belt and loads them into the chamber, making it as hard to add a suppressor as an AR-15, or just about any other gun, with the clear exception of revolvers, and even then there are some exceptions to that.

Reply:
Oh, and to burst a suppressor like that, you need to put about 500 rounds through it without a break, something quite literally impossible right now.

Reply:
Actually look up videos of the gun firing, even with a suppressor the people firing the weapon are still wearing ear protection. they only work fine in the sense that you can attach them.

Reply:
Almost all videos are of people at shooting ranges where other people may still be fireing unspressed guns, in addition, it's still proper safety requirements to wear hearing protection.

Reply:
Are we really arguing about modern day gun suppression for thousands of years in the future? Don't tell me it's impossible with space tech to suppress an ancient ballistic weapon.

Reply:
It's called sticking a potato at the end of your barrel. Which by the way should be a thing if it isn't already